composer.json

```diff
@@ -34,7 +34,8 @@
         "symfony/console":  "~3.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "~5.0"
+        "phpunit/phpunit":    "~5.0",
+        "mikey179/vfsStream": "^1.6"
```

Comment:
I would suggest doing that, or simply writing the file to a temporary location before reading it.

Using `vfsStream` for the tiny edge case surely feels like too much to me, but I'll let @maglnet put his word on it first.